common-arcana.lic

```diff
@@ -341,4 +348,12 @@ module DRCA
 
     cast?(data['cast'], data['symbiosis'], data['before'], data['after'])
   end
+
+  def check_to_harness(should_harness)
+    return false unless should_harness.equals? 'true'
+    return false unless DRSkill.getxp('Attunement') < DRSkill.getxp('Arcana')
```

Comment:
Do you think `return false if DRSkill.getxp('Attunement') >= DRSkill.getxp('Arcana')` would be clearer?

Reply:
These are clearer, but throughout this app we have unless and I've had it suggested to use unless over if quite a few times, lol. I don't care either way, it's the same thing to me.

Reply:
If is standard, so it's always clearer, I think. Unless is if not.

Reply:
I prefer `if` whenever possible.

That said, I prefer `unless` to `if !...`

Reply:
Gotcha, I'll remember that for the future for sure.